dart/tool/dart/generate_dart.dart

```diff
@@ -451,6 +456,30 @@ Stream<Event> onEvent(String streamName) => _getEventController(streamName).stre
     types.where((t) => !t.skip).forEach((t) => t.generate(gen));
   }
 
+  void _mergeTypes() {
+    final map = {};
```

Comment:
Can you add types here on the LHS to match the surrounding code? (`<String, Type>` ?)